pkg/asset/installconfig/openstack/openstack.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/openshift/installer/pkg/types/openstack"
 )
 
+const (
```

Comment:
I'm not sure why you need to do this, instead of just checking for an empty string

Reply:
Where is it being set to "<none>"?

Reply:
What happens if a user needs to create a custom install config? we enforce that empty string, or simply excluding the field is equivalent to not setting the value there. You should keep it consistent if possible

Reply:
Because the value returned by the menu gui is actually "<none>" , which is the choice presented to user.